src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -50,6 +51,35 @@ public static Envelope envelope(String jurisdiction, Status status) throws Excep
             timestamp,
             timestamp,
             UUID.randomUUID() + ".zip",
+            Classification.EXCEPTION,
+            false,
+            scannableItems(),
+            payments(),
+            nonScannableItems()
+        );
+
+        envelope.setStatus(status);
+        envelope.setContainer("SSCS");
+
+        return envelope;
+    }
+
+    public static Envelope envelopeUrgent() throws Exception {
+        return envelopeUrgent("SSCS", Status.CREATED);
+    }
+
+    public static Envelope envelopeUrgent(String jurisdiction, Status status) throws Exception {
+        Timestamp timestamp = getTimestamp();
+
+        Envelope envelope = new Envelope(
+            "SSCSPO",
+            jurisdiction,
+            timestamp,
+            timestamp,
+            timestamp,
+            UUID.randomUUID() + ".zip",
```

Comment:
We should use proper format for filenames else looking at the test it might look like filename is a random uuid

Reply:
I have not made any changes to this (ever). I suspect the semantics of the test would be changed either way, even if what you suggest would help with documenting the code.